src/test/java/htsjdk/variant/variantcontext/writer/VariantContextWriterBuilderUnitTest.java

```diff
@@ -123,6 +122,26 @@ public void testSetOutputFile() throws IOException {
     }
 
     @Test
+    public void testDetermineOutputType() {
+        Assert.assertEquals(OutputType.VCF, VariantContextWriterBuilder.determineOutputTypeFromFile(this.vcf));
+        Assert.assertEquals(OutputType.BCF, VariantContextWriterBuilder.determineOutputTypeFromFile(this.bcf));
+        Assert.assertEquals(OutputType.VCF_STREAM, VariantContextWriterBuilder.determineOutputTypeFromFile(new File("/dev/stdout")));
```

Comment:
specifically, is it worth testing the various BLOCK_COMPRESSED_EXTENSIONS for vcf as is done above?